Assets/GlitchEscape/Scripts/Interaction/InteractablePortal.cs

```diff
@@ -55,6 +60,7 @@ public void OnPlayerExitInteractionRadius(Player player)
 
     public void OpenPortal()
     {
-        this.transform.parent.gameObject.SetActive(true);
+        portalCutscene.Resume();
+        // this.transform.parent.gameObject.SetActive(true);
     }
 }
```

Comment:
Player can keep moving during cutscenes -  freeze player while cutscene is happening.

Reply:
This is a know issue, however I don't really want touch other scripts in this testing stage. Could be easily solved later.